recruiter/tests.py

```diff
@@ -1,3 +1,46 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from django.db.models import QuerySet
 
-# Create your tests here.
+from recruiter.models import Company, Recruiter
+
+
+@pytest.fixture
+def create_company():
+    company = Company(name="test_company", description="test_company_description", website_url="test_company_web")
+    company.save()
+    return company
+
+
+@pytest.fixture()
+def example_user():
+    user = User.objects.create_user("test", "testPassword")
+    user.save()
+    return user
+
+
+@pytest.fixture
+def create_recruiter(create_company, example_user):
+    recruiter = Recruiter(user=example_user, name="test_recruiter", company=create_company, email="test@email",
+                          phone_number="test_phone")
+    recruiter.save()
+    return recruiter
+
+
+@pytest.mark.django_db
+def test_get_all_companies_returns_all_companies_as_a_query_set(create_company):
+    companies_set = Company.get_all_companies()
+    assert isinstance(companies_set, QuerySet)
+    assert all(isinstance(company, Company) for company in companies_set)
+    assert list(companies_set.values_list("name", "description", "website_url")) == [
+        (create_company.name, create_company.description, create_company.website_url)]
+
+
+@pytest.mark.django_db
+def test_get_all_recruiters_returns_all_recruiters_as_a_query_set(create_recruiter, create_company):
```

Comment:
same comment as above - if you are passing fixtures to a test as parameters you are supposed to use them - some tools like pylint will actually raise an error for you code if you don't.